src/TodoApp/TodoApp.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import { TitleComponent } from './components/Title/TitleComponent';
+import { AddTodoComponent } from './components/AddTodo/AddTodoComponent';
+import { TodoListComponent } from './components/TodoList/TodoListComponent';
+import { getTodos, saveTodos } from './helpers/store';
```

Comment:
I generally try to avoid `helper` folders, the issue with "utility" folders is that you end up throwing a lot of code over time into them and the code becomes less [cohesive](https://www.geeksforgeeks.org/software-engineering-coupling-and-cohesion/). You should always try to structure your code with some design in mind, so that related code is close together.